### PR TITLE
kendalldoescoding linktree

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1044"></a>

### CODE REVIEW

File: public/data/Kendall_Does_Coding.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Abdus Salam",
```
Comment (Panquesito7) [inline public/data/Kendall_Does_Coding.json:2]: I think the name here should be Kendall Pinto 😅 am I right?
```suggestion
  "name": "Kendal Pinto",
```

File: public/data/Kendall_Does_Coding.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/kendalldoescoding",
+      "icon": "github"
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "githlab"
```
Comment (Panquesito7) [inline public/data/Kendall_Does_Coding.json:20]: There's no GitLab icon. If you want GitLab to be added, open an issue [**here**](https://github.com/EddieHubCommunity/LinkFree/issues/choose), we discuss it, then we decide to add it or not. 🙂

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Kendall_Does_Coding.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json:2]: Lol, I just took it from his template and did the needed edits LOL

Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json:2]: Thanks

File: public/data/Kendall_Does_Coding.json (hunk 2 above)
Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json:20]: That's anyway the wrong spelling I put, I'll create a issue shortly.

Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json]: #1045

File: public/data/Kendall_Does_Coding.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/kendalldoescoding",
+      "icon": "github"
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "gitlab"
```
Comment (Panquesito7) [inline public/data/Kendall_Does_Coding.json:20]: The icon does not exist yet. For now, could you use another icon, such as `link`? Thanks.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Kendall_Does_Coding.json (hunk 3 above)
Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json:20]: Sure

Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json]: Fixed in 8d7baa9

File: public/data/Kendall_Does_Coding.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/kendalldoescoding",
+      "icon": "github"
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "https://about.gitlab.com/images/press/press-kit-icon.svg"
```
Comment (Panquesito7) [inline public/data/Kendall_Does_Coding.json:20]: That...won't work as excepted 😅 we don't use icons that way.
As I said above, you can use another one such as `link`.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Kendall_Does_Coding.json (hunk 4 above)
Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json:20]: still can't understand ya @Panquesito7 lol

Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json:20]: can you co-author the commit?

Comment (emmalearnscode) [reply public/data/Kendall_Does_Coding.json:20]: Or you can write “gitlab” and it’ll default to globe until the gitlab icon gets added.

File: public/data/Kendall_Does_Coding.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "Quiz",
+      "url": "https://kendalldoescoding.gq/quiz",
+      "icon": "https://cdn-icons.flaticon.com/png/512/3261/premium/3261259.png?token=exp=1643398341~hmac=30c8bd086994bd1a76b2810fefdf0b4a"
```
Comment (emmalearnscode) [inline public/data/Kendall_Does_Coding.json:25]: Please choose from the available icons in the readme

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
@KendallDoesCoding so, here in LinkFree, we have our own set of icons. @Panquesito7 said to add the *Link* icon. Just put `"icon": "link"`:
image

**Review by Panquesito7 — COMMENTED** [review]
You might want to rename your file to `KendallDoesCoding` without the `_`. It would look better, IMO. What do you think?

**Comment by KendallDoesCoding** [comment]
> "icon": "link"

I still don't understand, if I say "icon": "link" it will take the gitlab logo?

**Comment by KendallDoesCoding** [comment]
> You might want to rename your file to `KendallDoesCoding` without the `_`. It would look better, IMO. What do you think?

done

File: public/data/Kendall_Does_Coding.json (hunk 5 above)
Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json]: Or, I can use icon: link?

File: public/data/Kendall_Does_Coding.json (hunk 4 above)
Comment (KendallDoesCoding) [reply public/data/Kendall_Does_Coding.json]: > Or you can write “gitlab” and it’ll default to globe until the gitlab icon gets added.

Done! Hope this works! Will mark this has resolved.

File: public/data/KendallDoesCoding.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "gitlab"
+    },
+    {
+      "name": "Quiz",
+      "url": "https://kendalldoescoding.gq/quiz",
+      "icon": "https://cdn-icons.flaticon.com/png/512/3261/premium/3261259.png?token=exp=1643398341~hmac=30c8bd086994bd1a76b2810fefdf0b4a"
```
Comment (emmalearnscode) [inline public/data/KendallDoesCoding.json:25]: You can’t use a link here, please choose an icon from the list in the readme or ask for one to be added in an issue.

Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:25]: What will happen if I use icon: link?

Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:25]: It will show a globe?

**Comment by KendallDoesCoding** [comment]
> @KendallDoesCoding so, here in LinkFree, we have our own set of icons. @Panquesito7 said to add the _Link_ icon. Just put `"icon": "link"`: image

@CodingSpecies Thanks! I added gitlab, incase the icon every gets added but once this gets merged it will only show a globe if there's no gitlab icon. Requesting a re-review from you!

File: public/data/KendallDoesCoding.json (hunk 6 above)
Comment (emmalearnscode) [reply public/data/KendallDoesCoding.json:25]: It will show a link icon like the one in the readme. If you want a globe write icon: “globe”

Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:25]: Oh, now I know what link is.

Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json]: Fixed this in commit 83df8e2

**Review by krshdev — APPROVED** [review]

File: public/data/KendallDoesCoding.json [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "link"
+    },
+    {
+      "name": "Quiz",
+      "url": "https://kendalldoescoding.gq/quiz",
+      "icon": "globe"
+    }
```
Comment (krshdev) [inline public/data/KendallDoesCoding.json:26]:
```suggestion
    },
```

**Review by krshdev — CHANGES_REQUESTED** [review]

File: public/data/KendallDoesCoding.json (hunk 7 above)
Comment (KendallDoesCoding) [reply public/data/KendallDoesCoding.json:26]: Thanks, I copyed that from the last one and forgot to add the comma.

**Review by CodingSpecies — APPROVED** [review]
Thank you @KendallDoesCoding

**Review by emmalearnscode — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**